docs/build/groups.md

```diff
@@ -0,0 +1,618 @@
+---
+sidebar_label: Group chats
+sidebar_position: 4
+description: Learn how to start, list, and cache conversations with XMTP
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Manage group chats with XMTP
+
+![Status](https://img.shields.io/badge/Reference_implementation_status-Alpha-orange)
+
+Secure group chats are an important part of every messaging app. In this guide, we delve into the essentials of using XMTP for creating secure group chats. From the initial steps of starting a new group chat, listing and caching conversations for quick access, to advanced topics like managing group members and synchronizing message history data across devices.
+
+:::caution Group Chats are Per Installation
+Group chats in XMTP are specific to each installation. This means that while you will see your group chat conversations across different devices, you will not see the historical messages within those chats automatically. Each group chat's message history is tied to the device where it was started. Consequently, there is no message history synced across devices. When you sign in on a new device, you will be able to see existing group chat conversations but will only receive new messages going forward. This behavior is specific to group conversations.
+:::
+
+## Create a group chat
+
+Initiate a new group chat with a list of specified addresses. To create a group, the recipient must have already started their client at least once on the XMTP network.
+
+<Tabs groupId="groupchats">
+<TabItem value="rn" label="React Native" attributes={{className: "rn_tab "}}>
+
+```jsx
+const group = await client.conversations.newGroup([
+  walletAddress1,
+  walletAddress2,
+]);
```

Comment:
You could call out the new admin permissions we just shipped as well.

You can now pass a permissions field to newGroup that can take a enum of 
everyone_is_admin
creator_is_admin

Which either allows everyone to add and remove members or only the group creator to do that.